cli/azd/internal/scaffold/spec.go

```diff
@@ -98,27 +98,56 @@ type ServiceSpec struct {
 	Name string
 	Port int
 
-	Env map[string]string
+	EnvironmentVariableInformation EnvironmentVariableInformation // todo: merge EnvironmentVariableInformation and env
```

Comment:
In `Env`, the value is quoted.
In `EnvironmentVariableInformation`, the value is not quated. It will be quoted in resources.bicept file, no quote will make it easier to print in console.